### ISSUE TITLE
Don't use CSS animations without giving the user a way to turn them off

### ISSUE THREAD

**Issue by mattcarlotta** [mattcarlotta/portfolio-astro-vercel#15]
- Give the option of turning off animations using `@media (prefers-reduced-motion)`
  - This should be added via tailwind [prefers reduced motion](https://tailwindcss.com/docs/hover-focus-and-other-states#prefers-reduced-motion)
  - Or this should be added via inline or global style elements
